util/internal/integrationtest/mapper/orm/src/main/java/org/hibernate/search/util/impl/integrationtest/mapper/orm/OrmSetupHelper.java

```diff
@@ -24,20 +26,33 @@
 import org.hibernate.search.util.impl.integrationtest.common.rule.BackendConfiguration;
 import org.hibernate.search.util.impl.integrationtest.common.rule.BackendMock;
 import org.hibernate.search.util.impl.integrationtest.common.rule.BackendSetupStrategy;
-import org.hibernate.search.util.impl.integrationtest.common.stub.backend.BackendMappingHandle;
 import org.hibernate.search.util.impl.integrationtest.common.rule.MappingSetupHelper;
+import org.hibernate.search.util.impl.integrationtest.common.stub.backend.BackendMappingHandle;
 import org.hibernate.search.util.impl.integrationtest.mapper.orm.multitenancy.impl.MultitenancyTestHelper;
 
 public final class OrmSetupHelper
 		extends MappingSetupHelper<OrmSetupHelper.SetupContext, SimpleSessionFactoryBuilder, SessionFactory> {
 
+	static {
+		// we don't need a ServiceLoader using a general-purpose aggregated class loader,
+		// since we expect the service impl in the direct dependent test module.
+		ServiceLoader<OrmSetupHelperConfig> serviceLoader = ServiceLoader.load( OrmSetupHelperConfig.class );
+		Iterator<OrmSetupHelperConfig> iterator = serviceLoader.iterator();
+		if ( iterator.hasNext() ) {
+			OrmSetupHelperConfig next = iterator.next();
+			defaultCoordinationStrategyExpectations = next.coordinationStrategyExpectations();
+		}
+		else {
+			defaultCoordinationStrategyExpectations = CoordinationStrategyExpectations.defaults();
+		}
+	}
+
 	public static void defaultAutomaticIndexingStrategy(
 			CoordinationStrategyExpectations coordinationStrategyExpectations) {
 		OrmSetupHelper.defaultCoordinationStrategyExpectations = coordinationStrategyExpectations;
 	}
 
-	private static CoordinationStrategyExpectations defaultCoordinationStrategyExpectations =
-			CoordinationStrategyExpectations.defaults();
+	private static CoordinationStrategyExpectations defaultCoordinationStrategyExpectations;
```

Comment:
I think this can now be `final`?

```suggestion
	private static final CoordinationStrategyExpectations defaultCoordinationStrategyExpectations;
```

And maybe we should adjust the case accordingly (`DEFAULT_COORDINATION_STRATEGY_EXPECTATIONS`).

Reply:
Actually, let's merge as-is; I need to push a related change afterwards, I'll do the change there.